salt/modules/consul.py

```diff
@@ -2126,7 +2131,7 @@ def acl_delete(consul_url=None, **kwargs):
         ret['res'] = False
         return ret
 
-    function = 'acl/delete/{0}'.format(kwargs['id'])
+    function = 'acl/destroy/{0}'.format(kwargs['id'])
```

Comment:
i don't retrieve when this API was "delete" instead of "destroy"
this the first commit of api/acl.go https://github.com/hashicorp/consul/commit/5555e0eb9ba5a18a14a080ab6a08ba08ebf2af85
and it aws "destroy"